.eslintcache

```diff
@@ -0,0 +1 @@
+{"/home/keshan/lingvokot/lingvokot.github.io/src/components/Footer.js":{"size":1757,"mtime":1492705229254,"hashOfConfig":"3965ce17149c3ed309685dee62a22647","results":{"filePath":"/home/keshan/lingvokot/lingvokot.github.io/src/components/Footer.js","messages":[],"errorCount":0,"warningCount":0}},"/home/keshan/lingvokot/lingvokot.github.io/src/components/Screens/ForInvestors.js":{"size":1247,"mtime":1492702390783,"hashOfConfig":"3965ce17149c3ed309685dee62a22647","results":{"filePath":"/home/keshan/lingvokot/lingvokot.github.io/src/components/Screens/ForInvestors.js","messages":[],"errorCount":0,"warningCount":0}},"/home/keshan/lingvokot/lingvokot.github.io/src/index.js":{"size":730,"mtime":1492701474165,"hashOfConfig":"3965ce17149c3ed309685dee62a22647","results":{"filePath":"/home/keshan/lingvokot/lingvokot.github.io/src/index.js","messages":[],"errorCount":0,"warningCount":0}}}
```

Comment:
Oh, sorry, forgot to mention, that this file should be ignored by git (add it to `.gitignore`)